modules/oadp-backing-and-restoring-from-cluster-to-cluster.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/advanced-topics.adoc
+
+
+:_content-type: CONCEPT
+[id="oadp-backing-and-restoring-from-cluster-to-cluster_{context}"]
+= Backing up data from one cluster and restoring it to another cluster
+
+In general, you back up data from one {product-title} cluster and restore it on another {product-title} cluster in the same way you back up and restore data to the same cluster. However, there are some additional prerequisites and differences in procedure when you back up data from one {product-title} cluster and restore it on another.
+
+.Prerequisites
+
+* All relevant prerequisites for backing up and restoring on your platform (AWS, Nicrosoft Azure, etc.) described in the relevant section of this guide, especially for the Data Protection Application (DPA).
```

Comment:
I'm confused about this content. What are "The relevant section of this guide, especially for the Data Protection Application (DPA).? Is there missing content somewhere?